native/packages/react-native-bpk-component-banner-alert/stories.js

```diff
@@ -166,7 +166,7 @@ class BpkBannerAlertFadeDemo extends Component<
 
 storiesOf('react-native-bpk-component-banner-alert', module)
   .addDecorator(CenterDecorator)
-  .add('docs:banner-alerts', () => (
+  .add('Banner alerts', () => (
```

Comment:
It looks like you are removing the `docs:default` example here but on further inspection you are not ... perhaps moving the `docs:` stories to the top will help?

Reply:
yes absolutely, good catch!

on a side note, why is always banner alert creating problems? 😞